js/AudioWorkletGlobalScope.js

```diff
@@ -11,7 +11,7 @@ const kWorkletInputs = Symbol.for('node-web-audio-api:worklet-inputs');
 const kWorkletOutputs = Symbol.for('node-web-audio-api:worklet-outputs');
 const nameProcessorCtorMap = new Map();
 const paramDescriptorRegisteredMap = new Map();
-// const processorIdMap = new WeakMap(); // instance, uuid
+const workletId = 0; // TODO how to receive from the Rust side?
```

Comment:
I could use your help here @b-ma. The workletId is in `Napi(Offline)AudioContext.worklet_id()` and I need it here :)

Reply:
Ok I'll have a look

Reply:
Done!